src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -459,6 +459,9 @@
            ProcessFrameworkReferences not be able to restore for the given RID.  -->
       <BuildArgs Condition="'$(DotNetBuildSourceOnly)' != 'true' and '@(TransitiveRepositoryReference->AnyHaveMetadataValue('Identity', 'runtime'))' == 'true'">$(BuildArgs) /p:DotNetBuildTargetRidOnly=true</BuildArgs>
 
+      <!-- Only pass when enabled to reduce command line noise. -->
+      <BuildArgs Condition="'$(DotNetBuildTests)' == 'true' and '$(DoNotBuildTests)' != 'true'">$(BuildArgs) /p:DotNetBuildTests=true</BuildArgs>
```

Comment:
I do think the names are deceptively similar. I propose a name like `ExcludeFromBuildingTests`.

Reply:
This doesn't need to be in a target. I would put this here: https://github.com/dotnet/sdk/blob/b24c716fd317ad13a6c5c9c7b09be745376cec80/src/SourceBuild/content/repo-projects/Directory.Build.targets#L40

Reply:
Yeah, that's better - thanks!

Reply:
Fixed with https://github.com/dotnet/sdk/pull/46394/commits/51744b8f0db7a7e446fdf678cf161a74607929c7